### PR TITLE
Restructure and reskin UI

### PR DESCRIPTION
**Pull request by peterwmwong**
@ddrscott Take a look at the `client/README.md`.  This PR allows a nice dev workflow that uses the webpack dev server for auto building/reloading while proxing API requests to the rails server.
The production build, is it a bit odd but works.  `sql_probe/live` has 2 endpoints (index, main) that statically serve the html, js, and css... and somehow magically works.  I'm sure there's a better way.

Also, this includes some nice UI updates since I last showed you yesterday, have a look:
https://www.dropbox.com/s/isoz8355gbkwfjt/2017-04-28%2001_44_43.mp4?dl=0

### CODE REVIEW

File: app/views/layouts/sql_probe/sql_probe.html.erb [hunk 1]
```diff
@@ -1,23 +1 @@
...
-  <%= yield :head %>
-</head>
-<body>
-
-<%= render 'layouts/sql_probe/navbar' unless @disable_top_navbar %>
-
 <%= yield %>
-
-<%= yield :javascript %>
-<%= render 'layouts/sql_probe/ga' if Rails.env.production? %>
```
Comment (peterwmwong) [inline app/views/layouts/sql_probe/sql_probe.html.erb]: Move this into react client

**Comment by ddrscott** [comment]
Note to self: Make spec/dummy use Puma so we can support websockets.

**Comment by ddrscott** [comment]
Add Procfile to run node server and Rails server at the same time.

File: app/views/layouts/sql_probe/sql_probe.html.erb (hunk 1 above)
Comment (ddrscott) [reply app/views/layouts/sql_probe/sql_probe.html.erb]: I see what you did. And did things more Railsy. Commit: 0eccf09

Basically, I put back the layout page, and told the live controller to ignore it.

**Comment by peterwmwong** [comment]
@ddrscott The client dev server now boots up on 3002.  I good with this PR going whenever you are good with it.  Let me know if there's anything we need to do before landing.

**Pull request merged**